src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -589,26 +596,13 @@ public function get_theme_item( $request ) {
 	 * Checks if a given request has access to read a single theme global styles config.
 	 *
 	 * @since 6.0.0
+	 * @since 6.7.0 Allow users with edit post capabilities to view theme global styles.
 	 *
 	 * @param WP_REST_Request $request Full details about the request.
 	 * @return true|WP_Error True if the request has read access for the item, WP_Error object otherwise.
 	 */
 	public function get_theme_items_permissions_check( $request ) {
```

Comment:
Can we deprecate this function as it uses `get_theme_item_permissions_check`?

Reply:
@mukeshpanchal27 There's some open tickets to extend the site editor post type permissions so I reckon keep it for now so we don't end up with deprecation followed by undeprecation.

Reply:
Could you please share it?

Reply:
https://core.trac.wordpress.org/ticket/54516